api/include/SAM_Battery.h

```diff
@@ -2399,6 +2399,8 @@ extern "C"
 
 	SAM_EXPORT double* SAM_Battery_Outputs_batt_dispatch_sched_mget(SAM_table ptr, int* nrows, int* ncols, SAM_error *err);
 
+	SAM_EXPORT double SAM_Battery_Outputs_batt_grid_charge_percent_nget(SAM_table ptr, SAM_error *err);
```

Comment:
These outputs are from https://github.com/NREL/SAM/pull/1865 and https://github.com/NREL/ssc/pull/1215. Sorry for not updating the autogen!